src/main/java/seedu/address/ui/TravelPlanPanel.java

```diff
@@ -39,11 +46,16 @@ public TravelPlanPanel(Directory directory) {
     public void update() {
         if (directory instanceof TravelPlan) {
             TravelPlan travelPlan = (TravelPlan) directory;
+            String cost = travelPlan.getTotalCost();
```

Comment:
Now no need do typecasting already. You can add this getTotalCost() method into Directory class.

Reply:
But still need to check isTravelPlan() to setText lah

Reply:
ok solved